cpp/ql/src/Documentation/CaptionedComments.qll

```diff
@@ -4,6 +4,10 @@
 
 import cpp
 
+/**
+ * Returns a string representation of the comment `c` containing the caption 'TODO' or 'FIXME'.
```

Comment:
```suggestion
 * Gets a string representation of the comment `c` containing the caption 'TODO' or 'FIXME'.
```

We prefer to avoid "returns". See https://wiki.semmle.com/pages/viewpage.action?pageId=74613282.